AnnoDesigner.Core/DataStructures/QuadTree.cs

```diff
@@ -42,18 +42,19 @@ public class Quadrant
             /// <summary>
             /// A count of all items in this Quadrant and under it.
             /// </summary>
-            int count;
+            private int count;
 
             /// <summary>
-            /// A list of all the items in the Quadrant and under it.
+            /// A list of all the items in the Quadrant and the items under it.
             /// </summary>
-            IEnumerable<(T Item, Rect Bounds)> itemCache;
+            private IEnumerable<(T Item, Rect Bounds)> itemCache;
 
             /// <summary>
             /// Holds a list of all the items in this quadrant.
             /// </summary>
             /// <remarks>
-            /// Stored as a list as items as any items that overlap multiple child quadrants will be stored here.
+            /// Stored as a list of items as any item that overlaps multiple child quadrants will be stored here.
+            /// This list differs to the itemCache, as this only contains items that do not fit completely into a child quadrant.
```

Comment:
So `Items` could be `ItemsInQuadrant` and `itemCache` could be `itemsInQuadrantIncludingChildren`?